doc/ale-idris.txt

```diff
@@ -0,0 +1,10 @@
+===============================================================================
+ALE Idris Integration                                   *ale-idris-options*
```

Comment:
Align the help tags to the right margin. If you use `:ALEFixSuggest`, it will suggest a function for fixing this.